### DISCUSSION TITLE
すみません、すみません。Qiita CLIが動きましたので。

### DISCUSSION THREAD

**Discussion by fygar256** [increments/qiita-discussions#913] [ヘルプ] [answered]
Qiita CLIが動きました。Spinning Wheel中なもので、直接webに投稿して、バグのあるプログラムを流してしまって、すみません。
お詫び申し上げます。これからはなるべくCLIを使って投稿しますので、バグが少なくなると思います。QiitaCLIにはまだ不慣れですけど、使っていって慣れようと思います。

**Comment by tanaka-asami415** [comment, marked as answer]
@fygar256 
CLIをご活用いただき、ありがとうございます。
ご不明な点がございましたら、お気軽にお問い合わせください！

**Reply by fygar256** [reply]
ご回答どうもありがとうございました。

**Comment by fygar256** [comment]
close
